lib/main.js

```diff
@@ -8,7 +8,7 @@ var dotenvExpand = function (config) {
       var key = match.replace(/\$|{|}/g, '')
 
       // process.env value 'wins' over .env file's value
-      var variable = process.env[key] || config.parsed[key] || ''
+      var variable = process.env.hasOwnProperty(key) ? process.env[key] : (config.parsed[key] || '')
```

Comment:
Should the two lines you updated be abstracted into a method given the similarities between the two?

Reply:
sorry, this is not code refactoring PR. it just fixing an issue. 
you can make you own fork and make some additional changes, but it looks like there no chance to merge it.